src/shards/JLLEnvs.jl

```diff
@@ -8,12 +8,18 @@ include("utils.jl")
 
 const JLL_ENV_SHARDS = Dict{String,Any}()
 
+const ARTIFACT_TOML_PATH = Ref{String}()
+
+# The platform used in Artifacts.toml
+const BINARY_BUILDER_PLATFORM = Base.BinaryPlatforms.Platform("x86_64", "linux"; libc="musl")
+
```

Comment:
```suggestion
```

IIRC, there is an "Any" platform that can be used if we do need to specify `platform` in the future.

Reply:
No, subtypes of `AbstractPlatform` are only `Platfrom`, `UnknownPlatform` and the ones defined in Pkg, and `UnknownPlatform` does not match any platforms.